compiler/embedded-import-value-test/test.lst

```diff
@@ -0,0 +1,190 @@
+#addeval(GatherNd_000)
+#addeval(Greater_000)
+#addeval(GreaterEqual_000)
+addeval(If_000)
+addeval(If_001)
+addeval(L2Normalize_000)
+addeval(L2Pool2D_000)
+#addeval(L2Pool2D_U8_000)
+addeval(LeakyRelu_000)
+#addeval(Less_000)
+#addeval(LessEqual_000)
+addeval(LocalResponseNormalization_000)
+#addeval(Log_000)
+#addeval(LogicalAnd_000)
+#addeval(LogicalNot_000)
+#addeval(LogicalOr_000)
+addeval(Logistic_000)
+#addeval(LogSoftmax_000)
+#addeval(MatMul_000)
+#addeval(MatrixDiag_000)
+#addeval(MatrixSetDiag_000)
+#addeval(Maximum_000)
+addeval(MaxPool2D_000)
+addeval(MaxPool2D_U8_000)
+addeval(Mean_000)
+addeval(Mean_001)
+#addeval(Mean_U8_000)
+#addeval(Minimum_000)
+#addeval(MirrorPad_000)
+addeval(Mul_000)
+#addeval(Mul_U8_000)
+#addeval(Neg_000)
+#addeval(NotEqual_000)
+#addeval(OneHot_000)
+#addeval(OneHot_001)
+#addeval(OneHot_002)
+#addeval(OneHot_003)
+#addeval(Pack_000)
+#addeval(Pack_U8_000)
+addeval(Pad_000)
+addeval(Pad_U8_000)
+#addeval(Pow_000)
+#addeval(PRelu_000)
+#addeval(Range_000)
+#addeval(Rank_000)
+#addeval(ReduceAny_000)
+#addeval(ReduceAny_001)
+#addeval(ReduceAny_002)
+#addeval(ReduceAny_003)
+#addeval(ReduceMax_000)
+#addeval(ReduceMin_000)
+#addeval(ReduceProd_000)
+#addeval(ReduceProd_001)
+#addeval(ReduceProd_002)
+#addeval(ReduceProd_003)
+#addeval(ReLU_000)
+#addeval(ReLU6_000)
+#addeval(ReLUN1To1_000)
+addeval(Reshape_000)
+addeval(Reshape_001)
+addeval(Reshape_002)
+#addeval(Reshape_003)
+addeval(Reshape_U8_000)
+#addeval(ResizeBilinear_000)
+#addeval(ResizeNearestNeighbor_000)
+#addeval(ReverseSequence_000)
+#addeval(ReverseV2_000)
+#addeval(Round_000)
+#addeval(Rsqrt_000)
+#addeval(ScatterNd_000)
+#addeval(SegmentSum_000)
+#addeval(Select_000)
+#addeval(Select_001)
+#addeval(Select_002)
+#addeval(SelectV2_000)
+#addeval(SelectV2_001)
+#addeval(SelectV2_002)
+#addeval(Shape_000)
+addeval(SignatureDef_MultiOut_000)
+addeval(SignatureDef_MultiOut_001)
+#addeval(Sin_000)
+addeval(Slice_000)
+addeval(Softmax_000)
+#addeval(Softmax_U8_000)
+#addeval(SpaceToBatchND_000)
+#addeval(SpaceToBatchND_001)
+#addeval(SpaceToBatchND_002)
+#addeval(SpaceToBatchND_003)
+addeval(SpaceToDepth_000)
+#addeval(SparseToDense_000)
+addeval(Split_000)
+#addeval(SplitV_000)
+#addeval(Sqrt_000)
+#addeval(Square_000)
+#addeval(SquaredDifference_000)
+addeval(Squeeze_000)
+addeval(Squeeze_001)
+addeval(StridedSlice_000)
+addeval(StridedSlice_001)
+addeval(StridedSlice_002)
+#addeval(Sub_000)
+#addeval(Sub_U8_000)
+#addeval(Sum_000)
+#addeval(Sum_001)
+#addeval(Tanh_000)
+#addeval(Tile_000)
+#addeval(Tile_U8_000)
+#addeval(TopKV2_000)
+#addeval(TopKV2_001)
+addeval(Transpose_000)
+#addeval(TransposeConv_000)
+addeval(Unpack_000)
+addeval(Unpack_001)
+addeval(Unpack_002)
+addeval(Unpack_003)
+#addeval(Where_000)
+#addeval(Where_001)
+#addeval(While_000)
+#addeval(While_001)
+#addeval(While_002)
+#addeval(While_003)
+#addeval(YUV_TO_RGB_U8_000)
+#addeval(ZerosLike_000)
+
+# Simple Network test
+addeval(Part_While_000)
+addeval(Part_While_001)
```

Comment:
List copied from https://github.com/Samsung/ONE/blob/73aeb8d1367d0aa8e3781f152bb6452e07c4365e/compiler/luci-value-test/test.lst.

Reply:
Why are most of them comment with `#` ?

Reply:
I just copied this file from `luci-value-test`, but as far as I know this tests can't be proceeded because not supported by `luci-interpreter`. I tried to uncomment this lines locally and obtain fail of following tests:

<details>
<summary> Failed tests </summary>

```
FAILED
- Abs_000
- AddN_000
- ArgMin_000
- ArgMin_001
- ArgMin_002
- ArgMin_003
- ArgMin_U8_000
- ArgMin_U8_001
- ArgMin_U8_002
- ArgMin_U8_003
- BatchMatMul_000
- BatchMatMulV2_000
- BatchMatMulV2_001
- Ceil_000
- Cos_000
- ExpandDims_000
- ExpandDims_001
- ExpandDims_002
- ExpandDims_003
- Fill_000
- Fill_001
- FloorMod_000
- FloorMod_001
- FullyConnected_U8_000
- GatherNd_000
- L2Pool2D_U8_000
- Log_000
- MatMul_000
- MatrixDiag_000
- MatrixSetDiag_000
- Mul_U8_000
- OneHot_003
- Range_000
- Rank_000
- ReduceAny_000
- ReduceAny_001
- ReduceAny_002
- ReduceAny_003
- ReduceMax_000
- ReduceMin_000
- ReduceProd_000
- ReduceProd_001
- ReduceProd_002
- ReduceProd_003
- ReLUN1To1_000
- Reshape_003
- ReverseSequence_000
- ReverseV2_000
- Round_000
- ScatterNd_000
- SegmentSum_000
- Select_000
- Select_001
- Select_002
- SelectV2_000
- SelectV2_001
- SelectV2_002
- Shape_000
- Sin_000
- SparseToDense_000
- Sum_000
- Sum_001
- Tile_000
- Tile_U8_000
- TopKV2_000
- TopKV2_001
- Where_000
- Where_001
- While_003
- ZerosLike_000
```

</details>

Log of this tests reports corresponding models contains unsupported operators =(

```
++ /home/mbronnikov/Work/ONE/build/compiler/embedded-import-value-test/test_driver --model /home/mbronnikov/Work/ONE/build/compiler/common-artifacts/AddN_000.circle
ERROR: Unsupported operator: 2 ofm
```

Reply:
From other hand not all commented cases unsupported. After `luci-interpreter` improvement some models are supported (e.g. Cast, OneHot). So I think better to enable this tests here and in https://github.com/Samsung/ONE/blob/73aeb8d1367d0aa8e3781f152bb6452e07c4365e/compiler/luci-value-test/test.lst.